pp/integrations/plugin-packs/procedures/network-switchs-hp-procurve-snmp.md

```diff
@@ -237,89 +254,100 @@ yum install centreon-plugin-Network-Switchs-Hp-Procurve-Snmp
+| WARNINGPORTSTATUS    | Define the conditions to match for the status to be WARNING. You can use the following variables: %{admin\_status}, %{oper\_status}, %{display}  |                                                       |             |
+| EXTRAOPTIONS         | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options).           | --verbose                                             |             |
 
 </TabItem>
 <TabItem value="Traffic-Generic-Id" label="Traffic-Generic-Id">
 
-| Macro        | Description                                                                                                                                | Default value     | Mandatory   |
-|:-------------|:-------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
-| INTERFACEID  | Set the interface (number expected) example: 1,2,... (empty means 'check all interfaces')                                                  |                   |             |
-| CRITICALIN   | Thresholds. : 'input-power' (dBm), 'bias-current' (mA), 'output-power' (dBm), 'voltage' (mV), 'module-temperature' (C), 'poe-power-actual' | 90                |             |
-| WARNINGIN    | Thresholds. : 'input-power' (dBm), 'bias-current' (mA), 'output-power' (dBm), 'voltage' (mV), 'module-temperature' (C), 'poe-power-actual' | 80                |             |
-| CRITICALOUT  | Thresholds. : 'input-power' (dBm), 'bias-current' (mA), 'output-power' (dBm), 'voltage' (mV), 'module-temperature' (C), 'poe-power-actual' | 90                |             |
-| WARNINGOUT   | Thresholds. : 'input-power' (dBm), 'bias-current' (mA), 'output-power' (dBm), 'voltage' (mV), 'module-temperature' (C), 'poe-power-actual' | 80                |             |
-| EXTRAOPTIONS | Any extra option you may want to add to the command (e.g. a --verbose flag). All options are listed [here](#available-options)                                                |                   |             |
+| Macro        | Description                                                                                                                            | Default value     | Mandatory   |
+|:-------------|:---------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| INTERFACEID  | Set the interface (number expected) example: 1,2,... (empty means 'check all interfaces')                                              |                   |             |
+| CRITICALIN   | Threshold                                                                                                                              | 90                |             |
```

Comment:
some info has been lost?

Reply:
En fait non c'était pas juste de mettre toutes les vaeurs ça vient du help compressé : 

```
=item B<--warning-*> B<--critical-*>

Thresholds.
Can be: 'total-port', 'total-admin-up', 'total-admin-down', 'total-oper-up', 'total-oper-down',
'in-traffic', 'out-traffic', 'in-error', 'in-discard', 'out-error', 'out-discard',
'in-ucast', 'in-bcast', 'in-mcast', 'out-ucast', 'out-bcast', 'out-mcast',
'speed' (b/s).

And also: 'input-power' (dBm), 'bias-current' (mA), 'output-power' (dBm), 'voltage' (mV), 'module-temperature' (C), 'poe-power-actual'.
```

Reply:
la commande de pp-imp c'est : 
`$CENTREONPLUGINS$/centreon_hp_procurve.pl --plugin=network::hp::procurve::snmp::plugin --mode=interfaces --hostname=$HOSTADDRESS$ --snmp-version='$_HOSTSNMPVERSION$' --snmp-community='$_HOSTSNMPCOMMUNITY$' $_HOSTSNMPEXTRAOPTIONS$ --interface='$_SERVICEINTERFACEID$' --add-status --add-traffic --warning-in-traffic='$_SERVICEWARNINGIN$' --critical-in-traffic='$_SERVICECRITICALIN$' --warning-out-traffic='$_SERVICEWARNINGOUT$' --critical-out-traffic='$_SERVICECRITICALOUT$' $_SERVICEEXTRAOPTIONS$`

Donc :
--warning-in-traffic='$_SERVICEWARNINGIN$' 
--critical-in-traffic='$_SERVICECRITICALIN$

donc WARNINGIN = warning-in-traffic seulement pas la liste proposée

Reply:
Je t'ai fait une proposition pour que ce soit plus détaillé mais juste